lib/core/test/integration/Test/Integration/Scenario/CLI/Server.hs

```diff
@@ -0,0 +1,63 @@
+module Test.Integration.Scenario.CLI.Server
+    ( spec
+    ) where
+
+import Prelude
+
+import Cardano.Launcher
+    ( Command (..), StdStream (..))
+import Test.Hspec.Expectations.Lifted
+    ( shouldBe )
+import Control.Monad
+    ( when )
+import System.Directory
+    ( doesFileExist, removePathForcibly )
+import Test.Hspec
+    ( SpecWith, after_, describe, it )
+import Test.Integration.Framework.DSL
+    ( Context (..), expectCmdStarts )
+
+spec :: SpecWith (Context t)
+spec = after_ tearDown $ do
+    describe "SERVER - cardano-wallet server" $ do
+        it "SERVER - Can start cardano-wallet server without --database" $ \_ -> do
+            let cardanoWalletServer = Command
+                    "cardano-wallet"
+                    [ "server"
+                    , "--random-port"
+                    ] (return ())
+                    Inherit
+            expectCmdStarts cardanoWalletServer
+
+            w1 <- doesFileExist dbFile
```

Comment:
```suggestion
doesFileExist dbFile `shouldReturn` False
```

Reply:
that was my first choice but:
```
/home/piotr/wb/cardano-wallet/lib/http-bridge/test/integration/Cardano/LauncherSpec.hs:46:71: error:
    • Couldn't match expected type ‘IO Bool’ with actual type ‘Bool’
    • In the second argument of ‘shouldBe’, namely ‘True’
      In a stmt of a 'do' block:
        (doesFileExist (stateDir ++ "/wallet.db-wal")) `shouldBe` True
      In the second argument of ‘($)’, namely
        ‘do let cardanoWalletLauncher
                  = Command "cardano-wallet-launcher" ... (return ()) Inherit
            expectCmdStarts cardanoWalletLauncher
            d1 <- doesDirectoryExist stateDir
            d2 <- doesDirectoryExist (stateDir ++ "/testnet")
            ....’
   |        
46 |             (doesFileExist (stateDir ++ "/wallet.db-wal")) `shouldBe` True
```

Reply:
I am not sure to get your point here :sweat_smile:  ? You example uses `shouldBe`, whereas I suggested `shouldReturn` (exactly because `shouldBe` works with raw / unwrapped values, but `shouldReturn` works with values inside a `Monad`).

Reply:
ooh, gotcha! (note to self: don't respond to comments after midnight, or... at least, proof read the next day...)